ludwig/trainers/trainer_lightgbm.py

```diff
@@ -479,7 +479,7 @@ def _construct_lgb_params(self) -> Tuple[dict, dict]:
                 output_params = {
                     "objective": "multiclass",
                     "metric": ["multi_logloss"],
-                    "num_class": feature.num_classes,
+                    "num_class": feature.decoder_config.num_classes,
```

Comment:
I believe num_classes should likely be a property of the output feature, not the decoder, as here for the lightgbm trainer it should not be required to provide a decoder config to begin with. @jppgks to doublecheck

Reply:
Awaiting @jppgks reply to determine next steps!

Reply:
Agree with @w4nderlust

Reply:
by default, GBM uses the passthrough decoder https://github.com/ludwig-ai/ludwig/blob/4405ef8745aab886e973dcdf3ad77c2048bf9f74/ludwig/utils/defaults.py#L313